code-scanning/phpmd.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# PHPMD is a spin-off project of PHP Depend and 
+# aims to be a PHP equivalent of the well known Java tool PMD.
+# More details at https://phpmd.org/
```

Comment:
Can we have more details here? Like what does it check for, why can this be useful for the user, etc?

Reply:
Sounds good, I have added the description from the home page of the tool to here. Please review.

Reply:
@shaopeng-gh Changes look good. One small change. Can you move the `What PHPMD does is...` below `PHPMD is a spin-off...`. That way users know what PHPMD is and what it does.